media/css/newsletter/newsletter-developer.scss

```diff
@@ -5,26 +5,40 @@
 @import '../pebbles/includes/lib';
 
 .section-intro {
-    background: url('/media/img/newsletter/developer/heart.svg') top 30px center no-repeat,
-                url('/media/img/newsletter/developer/grid.svg') top left repeat,
-                #0c99d5;
-    @include background-size(200px 199px, 60px 60px);
+    background: url('/media/img/newsletter/developer/Hearts.svg') top 30px center no-repeat,
+                url('/media/img/newsletter/developer/BG-Image.png') top left no-repeat,
+                #000000;
+
+    @include background-size(66px 44px, auto auto);
+    background-position: top 75px center, top center;
     height: 230px;
```

Comment:
It looks like the fixed height here is causing the text to be cut off on smaller screens (visible on locales that have longer strings, such as `/de/newsletter/developer/`). We may want to set a `min-height` instead and see how that works.

![copy-cropped](https://user-images.githubusercontent.com/400117/44274595-70daf200-a23a-11e8-93f3-963446ce522b.jpg)